### ISSUE TITLE
[Feature Request] Ability's to use Sony or XBOX trade

### ISSUE THREAD

**Issue by Joseph11Outlawz** [Vilsol/timeless-jewels#17]
Currently it appears that the tool only searches for timeless jewels on pc trade. Could you please create an option to select console trades instead.

**Comment by Vilsol** [comment]
@Joseph11Outlawz You can just switch to either console when you get redirected to the poe trade site.

**Comment by Joseph11Outlawz** [comment]
Yes you can, however it is not helpful when trying to find current jewels on the trade website. The tool is pointing to jewels that are not up for sale.
